### PR TITLE
docs: provide json logic examples

### LINKED ISSUE #26: Provide examples on how to use JSON Logic
Would be nice to see examples of how JSON logic be used in conjunction with EdgeX events.

### PR DESCRIPTION
**Pull request by rsdmike**
Provide some simple rules that can be applied to EdgeX.

Closes #26 

Signed-off-by: Mike <michael.johanson@intel.com>

### CODE REVIEW

File: app-services/json-logic/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# JSON Logic Examples
+
+This example demonstrates a few different ways to leverage JSON logic to assist in a bit more flexible and advanced way to perform filtering operations without the need to write your own functions. If you have many rules or want an even more feature rich rules engine check out [EMQ X Kuiper](https://github.com/emqx/kuiper) here.
+
+> Note: Only  operators that return true or false are supported. See http://jsonlogic.com/operations.html# for the complete list paying attention to return values. Any operator that returns manipulated data is currently not supported. 
+
+> Tip: Leverage http://jsonlogic.com/play.html to get your rule right before implementing in code. JSON can be a bit tricky to get right in code with all the escaped double quotes.
```
Comment (lenny-goodell) [inline app-services/json-logic/README.md:7]: We should add this tip to the App Function SDK documentation for JsonLogic function

File: app-services/json-logic/main.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+var counter int
+
+func main() {
+
+	// 1) First thing to do is to create an instance of the EdgeX SDK and initialize it.
+	edgexSdk := &appsdk.AppFunctionsSDK{ServiceKey: serviceKey}
+	if err := edgexSdk.Initialize(); err != nil {
+		message := fmt.Sprintf("SDK initialization failed: %v\n", err)
+		if edgexSdk.LoggingClient != nil {
```
Comment (lenny-goodell) [inline app-services/json-logic/main.go:46]: FYI, we no longer need to check LoggingClient . It is now always set first thing in Intialize.

File: app-services/json-logic/main.go [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+	// 2) shows how to access the application's specific configuration settings.
+	deviceNames, err := edgexSdk.GetAppSettingStrings("DeviceNames")
+	if err != nil {
+		edgexSdk.LoggingClient.Error(err.Error())
+		os.Exit(-1)
+	}
+	edgexSdk.LoggingClient.Info(fmt.Sprintf("Filtering for devices %v", deviceNames))
+	listOfDevices := strings.Join(deviceNames, "\",\"")
+
+	// Rule to look for a specific set of devices provided (similar to filterbydevicename)
```
Comment (lenny-goodell) [inline app-services/json-logic/main.go:63]: look => looks

**Review by lenny-goodell — APPROVED** [review]
Nice! Just a couple very minor things and a todo...

**Comment by Walt-Intel** [comment]
Looks good to me.

**Review by Walt-Intel — APPROVED** [review]
Looks good to me.

File: app-services/json-logic/main.go (hunk 2 above)
Comment (rsdmike) [reply app-services/json-logic/main.go:46]: i will fix this for all examples in a different PR

**Pull request merged**